README-FIPS.md

```diff
@@ -77,6 +81,78 @@ If you configured OpenSSL to be installed to a different location, the paths wil
 vary accordingly. In the rare case that you need to install the fipsmodule.cnf
 to non-standard location, you can execute the `openssl fipsinstall` command manually.
 
+Installing the FIPS provider and using it with the latest release
+=================================================================
+
+This normally requires you to download 2 copies of the OpenSSL source code.
+
+Download and build the (OpenSSL 3.0) FIPS provider
+--------------------------------------------------
+
+OpenSSL 3.0 is the only source that is currently FIPS validated.
+If you use any other source currently it is NOT FIPS valid.
+Please refer to the security guide for guidance related to
+the OpenSSL 3.0  FIPS provider.
+
+    $ wget https://www.openssl.org/source/openssl-3.0.0.tar.gz
+    $ tar -xf openssl-3.0.0.tar.gz
+    $ cd openssl-3.0.0
+    $ ./Configure enable-fips
+    $ make
+    $ cd ..
+
+Download and build the latest release of OpenSSL
+------------------------------------------------
+
+We use OpenSSL 3.1 here, (but you could also use the latest 3.0.X)
+
+    $ wget https://www.openssl.org/source/openssl-3.1.0.tar.gz
+    $ tar -xf openssl-3.1.0.tar.gz
+    $ cd openssl-3.1.0
+    $ ./Configure enable-fips
+    $ make -j 4
+
+Use the OpenSSL 3.0 FIPS provider for testing
+---------------------------------------------
+
+We do this by replacing the artifact for the OpenSSL 3.1 FIPS provider.
+Note that the OpenSSL 3.1 FIPS provider has not been validated
+so it must not be used for FIPS purposes.
+
+    $ cp ../openssl-3.0.0/providers/fips.so providers/.
+    $ cp ../openssl-3.0.0/providers/fipsmodule.cnf providers/.
+
+    // Validate the output of the following to make sure we are using the
+    // OpenSSL 3.0 FIPS provider
+    $ ./util/wrap.pl -fips apps/openssl list -provider-path providers \
+    -provider fips -providers
+
+    // Now run the current tests using the OpenSSL 3.0 FIPS provider.
+    $ make tests
+
+Copy the FIPS provider artifacts (fips.so & fipsmodule.cnf) to known locations
```

Comment:
Missing full stop.

Reply:
and (minor)
```suggestion
Copy the FIPS provider artifacts (`fips.so` & `fipsmodule.cnf`) to known locations
```

Reply:
Added the ticks - but not the full stop. They are section headings.